includes/commerce_pos.transaction.inc

```diff
@@ -259,6 +260,7 @@ function commerce_pos_transaction_form(&$form, &$form_state, $transaction_type)
         '#prefix' => '<div class="price-edit-wrapper line-item-col">',
         '#suffix' => '</div>',
       );
+      unset($currency);
```

Comment:
These unsets seem unnecessary, we don't really benefit in php from clearing out variables like we would in something like C++